CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-# 0.4.7-dev1
+# 0.4.7-dev2
```

Comment:
let's remove the `-dev2` since this will be the published `0.4.7`.

Reply:
Done!